apps/support/Makefile.inc

```diff
@@ -97,6 +97,7 @@ ANDROID_NDK_HOST_PLATFORM ?= $(shell echo `uname` | tr '[:upper:]' '[:lower:]')
 ANDROID_API_VERSION ?= 26
 
 CXX-host ?= $(CXX)
+CXX-host-profile ?= $(CXX)
```

Comment:
I don't think we should add -profile versions of these. It's not really a different target, and it's the start of a combinatorial hell hole (e.g. why not CXXFLAGS-host-opencl-profile? And what about CXXFLAGS-host-profile-opencl?)

Reply:
Eh, fair point.